lte/cloud/go/services/lte/servicers/builder_servicer_test.go

```diff
@@ -185,6 +185,31 @@ func TestBuilder_Build(t *testing.T) {
 	actual, err = build(&nw, &graph, "gw1")
 	assert.NoError(t, err)
 	assert.Equal(t, expected, actual)
+
+	// verify restricted plmns
+	setEpcNetworkRestrictedPlmns(&nw, []*lte_models.PlmnConfig{{Mcc: "100", Mnc: "010"}})
+	mmeVals := expected["mme"].(*lte_mconfig.MME)
+	mmeVals.RestrictedPlmns = []*lte_mconfig.MME_PlmnConfig{
+		{
+			Mcc: "100",
+			Mnc: "010",
+		},
+	}
+
+	actual, err = build(&nw, &graph, "gw1")
+	assert.NoError(t, err)
+	assert.Equal(t, expected, actual)
+
+	//verify service area map
+	setEpcNetworkServiceAreaMap(&nw, map[string]lte_models.TacList{"001": []lte_models.Tac{111, 112}})
+	mmeVals.ServiceAreaMaps = map[string]*lte_mconfig.MME_TacList{
+		"001": {
+			Tac: []uint32{111, 112},
+		},
```

Comment:
as a nit, maybe adding multiple entries in the test case would be better.

Reply:
+1 for this

Reply:
Done. Thanks for the review.